From 8bb98a8f65bedb4eae1ade333c771187660e45b6 Mon Sep 17 00:00:00 2001
From: Tom Marshall <tdm.code@gmail.com>
Date: Fri, 18 Jan 2019 09:41:18 -0800
Subject: [PATCH 4/4] e2fsprogs: Fix resize2fs_static build

Change "resize: Make a target for standalone resize2fs_static binary"
added a top level Android.mk which broke e2fsdroid_static because the
macro all-sudbir-makefiles only looks one level deep.

Fix this by adding an Android.mk to contrib, so that the build system
sees contrib/android/Android.mk.

Change-Id: I91e673dd207af3f6c7176370b3598a5eefd952b2
---
 contrib/Android.mk | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 contrib/Android.mk

diff --git a/contrib/Android.mk b/contrib/Android.mk
new file mode 100644
index 00000000..5053e7d6
--- /dev/null
+++ b/contrib/Android.mk
@@ -0,0 +1 @@
+include $(call all-subdir-makefiles)
-- 
2.17.1

